src/api/messages/escrow/EscrowLockMessage.ts

```diff
@@ -0,0 +1,24 @@
+import { IsNotEmpty } from 'class-validator';
+import { GenerateMessageInterface } from './../GenerateMessageInterface';
+
+export class EscrowLockMessage implements GenerateMessageInterface {
+
```

Comment:
theres allready ActionMessageInterface, use that.
you could create EscrowMessageInterface which implements ActionMessageInterface, or just make EscrowLockMessage implements ActionMessageInterface